sonar-python-plugin/src/main/java/org/sonar/plugins/python/PythonSquidSensor.java

```diff
@@ -97,6 +97,7 @@ public void execute(SensorContext context) {
 
     List<SquidAstVisitor<Grammar>> visitors = Lists.newArrayList(checks.all());
     visitors.add(new FileLinesVisitor(fileLinesContextFactory, context.fileSystem(), linesOfCode));
+    visitors.add(new PythonHighlighter(context));
```

Comment:
@ivandalbosco i think it would be nice to have at least one assertion about highlighting in PythonSquidSensorTest